ntoskrnl/mm/section.c

```diff
@@ -4579,11 +4579,11 @@ MmMapViewOfSection(IN PVOID SectionObject,
         ImageSectionObject->ImageInformation.ImageFileSize = (ULONG)ImageSize;
 
         /* Check for an illegal base address */
-        if (((ImageBase + ImageSize) > (ULONG_PTR)MmHighestUserAddress) ||
+        if (((ImageBase + ImageSize) > (ULONG_PTR)MM_HIGHEST_VAD_ADDRESS) ||
```

Comment:
No idea whether looking for ImageBase alone being > MM_HIGHEST_VAD_ADDRESS would make also sense...

Reply:
I have created a ticket for that: https://jira.reactos.org/browse/CORE-16456